iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/Replace3TwoWindingsTransformersByThreeWindingsTransformers.java

```diff
@@ -288,25 +290,26 @@ private static String getEnd1(boolean isWellOriented) {
         return isWellOriented ? "1" : "2";
     }
 
-    private static Map<Terminal, Terminal> getRegulatedTerminalMapping(TwoR twoR, ThreeWindingsTransformer t3w) {
-        Map<Terminal, Terminal> regulatedTerminalMapping = new HashMap<>();
-        regulatedTerminalMapping.put(getTerminal1(twoR.t2w1, isWellOriented(twoR.starBus, twoR.t2w1)), t3w.getLeg1().getTerminal());
-        regulatedTerminalMapping.put(getTerminal1(twoR.t2w2, isWellOriented(twoR.starBus, twoR.t2w2)), t3w.getLeg2().getTerminal());
-        regulatedTerminalMapping.put(getTerminal1(twoR.t2w3, isWellOriented(twoR.starBus, twoR.t2w3)), t3w.getLeg3().getTerminal());
-        return regulatedTerminalMapping;
+    private static void replaceRegulatedTerminal(ThreeWindingsTransformer.Leg t3wLeg, ThreeWindingsTransformer t3w, TwoR twoR) {
```

Comment:
The `t3w` parameter is not needed since it can be retrieved from the leg:
`t3wLeg.getTransformer()`

Reply:
Done

Reply:
Done